core/src/MainFrame/Model/SoundEffect.java

```diff
@@ -7,5 +7,9 @@
 	public static void playSoundEffect(SoundEffect x){
 		
 	}
+	
+	public static void playSound(SoundEffect x, boolean loop) {

```

Comment:
Apparently I'm blind, so just add "Effect" into the method header. Also rip me, Github refuses to play nice for some reason, sorry man.